docs/other_frameworks.md

```diff
@@ -322,17 +322,17 @@ export default {
 Usage is simple
 
 ```vue
-<form-group :validator="$v.form.email" label="Email with wrapper">
```

Comment:
Lets keep it form-group to be consistent with other examples. Just rename the `name` of the Component definition above.

Reply:
Alright, will edit the PR.

Reply:
Awesome :)